docs/automated-tests/test-execution/quick-execution.md

```diff
@@ -1,55 +1,57 @@
+Test Studio launches the selected browser on top of any other running apps and executes the recorded steps.
 
 ![Test Studio][4]
 
 > __Important__
 > <br>
 > <br>
-> While a test is being executed **do not start another instance of the same browser or any other application** until the run is finished!
+> Do not start another instance of the same browser or any other application until the test completes!
 
 ## Quick Run Results
 
-The <a href="/automated-tests/test-results/analyze-quick-run-results" target="_blank">results from the quick execution</a> mode are dedicated mainly for <a href="/automated-tests/troubleshooting/failing-test" target="_blank">debugging any inconsistencies</a> in the recorded test steps. Therefore, these are only __temporary results for the last initiated run__. These will be overridden when the test is executed again using the __Execute__ button, or deleted - if the test/project is closed and reopened.
+The <a href="/automated-tests/test-results/analyze-quick-run-results" target="_blank">results from the quick execution</a> mode are dedicated mainly to <a href="/automated-tests/troubleshooting/failing-test" target="_blank">debugging inconsistencies</a> in the recorded test steps. Therefore, these are only temporary results for the last test run. These results will be overwritten when you execute the test again or deleted if you close and reopen the test/project.
 
-Once the test run is finished, the Test Steps pane displays the overall result - whether the test is passed or failed, which step failed, if any. You can also access a complete execution log and the <a href="/automated-tests/test-results/step-failure-details" target="_blank">failure details</a> for the failing step, if applicable.
+Once the test completes, the __Test steps__ pane displays the overall result - passed or failed and any failed steps. You can also access a complete execution log and the <a href="/automated-tests/test-results/step-failure-details" target="_blank">failure details</a> for the failing step.
 
 ![Quick run results][3]
 
 > __Tip__
 > <br>
 > <br>
-> Once the test is adjusted and demonstrates consistent execution behavior, you can <a href="/automated-tests/test-lists/test-lists-standalone" target="_blank">include it in a test list</a>. The results generated from test list runs are being stored on project level.
+> Once you adjust the test and it demonstrates consistent execution behavior, you can <a href="/automated-tests/test-lists/test-lists-standalone" target="_blank">include it in a test list</a>. The results generated from test list runs are stored on project level.
 
 ## Options to Modify for the Quick Test Run
 
-Test Studio provides multiple options to ease you in executing the tests and debugging any encountered failures. The mostly used settings are added in the ___Quick Test Execution___ ribbon for quick access. Details for these quick access options can be found in the referenced articles below:
```

Comment:
I don't see a __Quick Test Execution__ ribbon. Do you mean __Tests__? Stay true to the names of the UI elements in the tool.